docs/updating/nim-migration/migration-to-new-installation-methods.md

```diff
@@ -317,15 +317,15 @@ What we will sunset on this date:
 
 To migrate your project to the new installation methods, you can follow the instructions below.
 
-If you maintain custom plugins for CKEditor&nbsp;5, you need to update them first.
+First, proceed with migrating your project, depending on the old installation method you are using.
 
-* {@link updating/nim-migration/custom-plugins Migrating custom plugins}
+* {@link updating/nim-migration/predefined-builds Migrating from predefined builds}.
+* {@link updating/nim-migration/customized-builds Migrating from customized builds}.
+* {@link updating/nim-migration/dll-builds Migrating from DLL builds}.
 
-After you have updated your custom plugins, you can proceed with migrating your project, depending on the old installation method you are using.
+Second, if you maintain any CKEditor&nbsp;5 custom plugins **and publish them to npm** for you need to update them as well:
```

Comment:
I changed the order as I consider custom libs on npm, a secondary action not the first one.

Reply:
It's not necessary to publish them to npm, but just having them as a separate npm package, for example in a monorepo.